en_US.json

```diff
@@ -244,6 +244,22 @@
 	"certificate.deploy.inprogress.modal.title": "Deploying Certificate",
 	"certificate.deploy.success.modal.message": "The certificate has been successfully deployed.",
 	"certificate.deploy.success.modal.title": "Certificate Deployed",
+	"certificate.dns01chanllenge.error.another.instance": "Our system is busy validating another request. Please try again in a few minutes.",
+	"certificate.dns01chanllenge.error.exist.certificate": "You have an existing certificate that has exactly the same domains or certificate name you requested and isn't close to expiry.",
+	"certificate.dns01chanllenge.error.exist.timeout": "Time out to wait. Please try again",
+	"certificate.dns01chanllenge.fail.above.message": "We are unable to validate the following DNS records:",
+	"certificate.dns01chanllenge.fail.below.message": "Ensure that you have created the DNS records. If you have, you may need to wait longer for the DNS changes to propagate.",
+	"certificate.dns01chanllenge.fail.info.modal.title": "It failed to authenticate some domains",
```

Comment:
What is meant by "authenticate" in this context?
Maybe "DNS records not found for some domains"  would be better?

Reply:
I fixed some text and changed the language for ar_SA.
Thanks